<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Sebastian Gibb1,2,✉, and Thorsten Kaiser1" />

<meta name="date" content="2022-06-19" />

<title>A New Model to Predict Survival in Patients with End-Stage Liver Disease</title>

<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="site_libs/viz-1.8.2/viz.js"></script>
<link href="site_libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="site_libs/grViz-binding-1.0.9/grViz.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>

<style type="text/css">
  p.abstract{
    text-align: center;
    font-weight: bold;
  }
  div.abstract{
    margin: auto;
    width: 90%;
  }
</style>


<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ampel-leipzig-meld</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Exploratory Data Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="tables.html">Tables</a>
    </li>
    <li>
      <a href="missing.html">Missing Data</a>
    </li>
    <li>
      <a href="histograms.html">Histograms</a>
    </li>
    <li>
      <a href="boxplots.html">Boxplots</a>
    </li>
    <li>
      <a href="dotcharts.html">Dotcharts</a>
    </li>
    <li>
      <a href="linecharts.html">Linecharts</a>
    </li>
    <li>
      <a href="corrplot.html">Correlation</a>
    </li>
    <li>
      <a href="survivalplots.html">Survival Plots</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Machine Learning
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="benchmarks.html">Benchmarks</a>
    </li>
    <li>
      <a href="elasticnet.html">ElasticNet</a>
    </li>
    <li>
      <a href="variableimportance.html">Variable Importance</a>
    </li>
    <li>
      <a href="roc.html">ROC</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Article
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="article.html">Article</a>
    </li>
    <li>
      <a href="supplement.html">Supplement</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="pipeline.html">Pipeline</a>
</li>
<li>
  <a href="https://github.com/ampel-leipzig/ampel-leipzig-meld">Source Code</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">A New Model to Predict Survival in Patients with End-Stage Liver Disease</h1>
<h3 class="subtitle"><p>Comparision against MELD, MELD-Na, MELD-Plus7.</p></h3>
<h4 class="author">Sebastian Gibb<sup>1,2,✉</sup>, and Thorsten Kaiser<sup>1</sup></h4>
<h4 class="date">2022-06-19</h4>
<div class="abstract">
<p class="abstract">Abstract</p>
<p><strong>Objectives</strong>:
The shortage in grafts for liver transplantation requires risk stratification and allocation rules.
This study aims to improve the model of end-stage liver disease (MELD) score for 90-day mortality prediction as an risk stratification tool with the help of machine learning (ML) algorithms.</p>
<p><strong>Methods</strong>:
We retrospectively analysed clinical and laboratory data of 654 patients who were recruited during the evaluation process for liver transplantation at the University Hospital Greifswald.
After comparing 13 different ML algorithms in a nested cross-validation setting and selecting the best performing one we build a new model to predict 90-day mortality in patients with end-stage liver disease.</p>
<p><strong>Results</strong>:
Penalized regression algorithms yield the highest prediction performance in our ML algorithm benchmark.
In favour of a simpler model we chose the least absolute shrinkage and selection operator (lasso) regression.
Beside the classical MELD variables INR and bilirubin the lasso regression selected
cystatin c over creatinine, and additionally IL-6, total protein and cholinesterase.
The new model offers a better discrimination and calibration than MELD and MELD with sodium (MELD-Na) or the MELD-Plus7 risk score.</p>
<p><strong>Conclusion</strong>:
We provide a new model of end-stage liver disease that outperforms the classical MELD score for 90-day survival prediction.</p>
<p><strong>Keywords</strong>:
end-stage liver disease; MELD; risk stratification; survival estimation; IL-6.</p>
</div>

</div>


<p><sup>1</sup> Institute of Laboratory Medicine, Clinical Chemistry and Molecular Diagnostics, University Hospital Leipzig, Paul-List-Str. 13-15, D-04103 Leipzig, Germany.<br />
<sup>2</sup> Anesthesiology and Intensive Care Medicine, University Hospital Greifswald, Ferdinand-Sauerbruch-Straße, D-17475 Greifswald, Germany.</p>
<p><sup>✉</sup> Correspondence: <a href="mailto:mail@sebastiangibb.de">Sebastian Gibb &lt;<a href="mailto:mail@sebastiangibb.de" class="email">mail@sebastiangibb.de</a>&gt;</a></p>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks">
Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-06-19
</p>
<p>
<strong>Checks:</strong>
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
7
<span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
0
</p>
<p>
<strong>Knit directory:</strong>
<code>ampel-leipzig-meld/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the
reproducibility checks that were applied when the results were created.
The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you
know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>Environment:</strong> empty
</a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global
environment can affect the analysis in your R Markdown file in unknown ways.
For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210604code">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>Seed:</strong> <code>set.seed(20210604)</code>
</a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210604code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210604)</code> was run prior to running the code in the R Markdown file.
Setting a seed ensures that any results that rely on randomness, e.g.
subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>Session information:</strong> recorded
</a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is
critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>Cache:</strong> none
</a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident
that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>File paths:</strong> relative
</a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project
makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomampelleipzigampelleipzigmeldtreeac8a09d21167c0d12ade7344b93dcc5f211ea00atargetblankac8a09da">
<span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
<strong>Repository version:</strong> <a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/tree/ac8a09d21167c0d12ade7344b93dcc5f211ea00a" target="_blank">ac8a09d</a>
</a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomampelleipzigampelleipzigmeldtreeac8a09d21167c0d12ade7344b93dcc5f211ea00atargetblankac8a09da" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and
connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/tree/ac8a09d21167c0d12ade7344b93dcc5f211ea00a" target="_blank">ac8a09d</a>.
See the <em>Past versions</em> tab to see a history of the changes made to the
R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the
analysis have been committed to Git prior to generating the results (you can
use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only
checks the R Markdown file, but you know if there are other scripts or data
files that it depends on. Below is the status of the Git repository when the
results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    TODO
    Ignored:    _targets/
    Ignored:    container/
    Ignored:    logs/
    Ignored:    scripts/R.sh

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in
this status report because it is ok for generated content to have uncommitted
changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made
to the R Markdown (<code>analysis/article.Rmd</code>) and HTML (<code>docs/article.html</code>)
files. If you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table below to
view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/396872946953ef952957f2a742e35bb99d3c1b61/analysis/article.Rmd" target="_blank">3968729</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-19
</td>
<td>
feat: cite ameld with zenodo doi
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/fb43d010446fb8613cf15ede3371bf2c236599d9/docs/article.html" target="_blank">fb43d01</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-19
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/abe89edafbb73fad7cb4132aceceedf9b52fb27d/analysis/article.Rmd" target="_blank">abe89ed</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-19
</td>
<td>
fix: cite zlog with zenodo doi
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/a10ff910fcef08570d65b6edf52df097cd3c3ce0/analysis/article.Rmd" target="_blank">a10ff91</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-16
</td>
<td>
fix: don’t echo code chunks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/ebe29cf9f65cea961c6ea141aa04514818137f8c/docs/article.html" target="_blank">ebe29cf</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-16
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/43a6574a29155822da611790404e17208038981a/analysis/article.Rmd" target="_blank">43a6574</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-16
</td>
<td>
feat: further work on the manuscript
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/803521940699fbefb650f4567d9d5872332097d1/docs/article.html" target="_blank">8035219</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-15
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/19d9fedc9c7b07996332c2ed9671ba03ad3191d7/analysis/article.Rmd" target="_blank">19d9fed</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-15
</td>
<td>
feat: further work on the manuscript
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/d3e94627fae586919bec7433b9d7c2c03b7d4e93/docs/article.html" target="_blank">d3e9462</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-06
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/3181fbec9ef43b704d7ddae53225636b899f3ac6/analysis/article.Rmd" target="_blank">3181fbe</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-06
</td>
<td>
refactor: suppress warning for table1
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/b20484abeb3ca434ad31f290a6f9531c8b40a16a/docs/article.html" target="_blank">b20484a</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-06
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/01f934b19920c5f4612f5ec2042502ce1c8a22ee/analysis/article.Rmd" target="_blank">01f934b</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-06
</td>
<td>
feat: add first discussion draft
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/13ac21ebda6467870c7a207c5697c0352518b10c/analysis/article.Rmd" target="_blank">13ac21e</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-06
</td>
<td>
feat: add docx export
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/983ec69e93a2b399ab7ff8e81ba04b5727cdacbb/docs/article.html" target="_blank">983ec69</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-03-17
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/85ff0714e9ff53f5332037f22e586c285ed21925/analysis/article.Rmd" target="_blank">85ff071</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-03-15
</td>
<td>
fix: co2eq calculation
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/839f920affe40c71e720f2049e94650476bcc4da/analysis/article.Rmd" target="_blank">839f920</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-03-11
</td>
<td>
fix: cite timeROC package
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/4a51c7d0450311dacd0c021d911b2683516d9a00/analysis/article.Rmd" target="_blank">4a51c7d</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-03-11
</td>
<td>
feat: add co2eq estimation
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/c5d6267aee95d714e59e2205f35024287c9d1a74/analysis/article.Rmd" target="_blank">c5d6267</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-03-10
</td>
<td>
feat: describe model development in methods section
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/74483a56f64adbcb0777db4edc3788e4da38f55f/analysis/article.Rmd" target="_blank">74483a5</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-02-10
</td>
<td>
chore: remove comments
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/09c3fab26d1f5e68061433609cb5ad957101b1f8/analysis/article.Rmd" target="_blank">09c3fab</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-02-10
</td>
<td>
refactor: switch to grid search
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/373e7d8009a9d27dd2b1ea789522d397ffbb7cbd/docs/article.html" target="_blank">373e7d8</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-20
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/8b3ba9c9aaa97e4553f4781f9cddc4163ad66a4b/analysis/article.Rmd" target="_blank">8b3ba9c</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-19
</td>
<td>
feat: cite gtsummary
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/28ad932ab9c0b5fac3d38a839cc331552bb819ad/analysis/article.Rmd" target="_blank">28ad932</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-19
</td>
<td>
feat: describe zlog in more detail
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/df8964f95a7d42c7f88649245d1a027ff8108cda/docs/article.html" target="_blank">df8964f</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-15
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/2697eadb3908db885f3d7667ada498b647eb37ff/analysis/article.Rmd" target="_blank">2697ead</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-15
</td>
<td>
fix: remove missing cross reference
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/eeca97608df7bc66036e27c9a73b26973f3f3b6e/analysis/article.Rmd" target="_blank">eeca976</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-10-15
</td>
<td>
feat: add ml method part
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/c66640c2b1856a5311392b2592f58ca80a5dfd34/docs/article.html" target="_blank">c66640c</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-09-14
</td>
<td>
feat: first nnet tests
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/afa48d91c0fd2263888b6b84aceb6e3c33e4209a/docs/article.html" target="_blank">afa48d9</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-07
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/d32f7b3a15ac6425ffdd29c3c9f65cea509d0bad/analysis/article.Rmd" target="_blank">d32f7b3</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-07
</td>
<td>
fix: gtsummary compact theme
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/ebf878baa794b00af4b5ed48ef6810939f69f8ed/analysis/article.Rmd" target="_blank">ebf878b</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-07
</td>
<td>
fix: observed vs expected mortality table
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/4ec3d4096c4b1188deb4108529389b4f94cd356f/analysis/article.Rmd" target="_blank">4ec3d40</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-04
</td>
<td>
feat: add table 1 and table observed vs expected mortality
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/e41dd8c1cb6570099ed69cf8769a83b67bd11c55/analysis/article.Rmd" target="_blank">e41dd8c</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-03
</td>
<td>
feat: add flowchart and discussion notes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/523c2860f3ee9161e3b07b1392de88a79d31ec9e/analysis/article.Rmd" target="_blank">523c286</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-02
</td>
<td>
refactor: use bookdown::html_document2 instead of workflowr::wflow_html to allow crossreferences
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/ampel-leipzig/ampel-leipzig-meld/3aab3e106752adffec84e36600afb9590d92004c/docs/article.html" target="_blank">3aab3e1</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-01
</td>
<td>
chore: rebuild site
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/91bfbd2316b656aa5aefc6c21ba97f7cb7af2297/analysis/article.Rmd" target="_blank">91bfbd2</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2021-08-01
</td>
<td>
feat: add first article draft with pandoc filters
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Liver cirrhosis is the terminal result of fibrotic remodeling
of liver tissue due to chronic damage. An organ failure is often
irreversible and the only available therapy is liver transplantation.
However, the shortage in grafts for transplantation from deceased donors
requires risk stratification and precise allocation rules.
The allocation of liver transplantation in most countries is based on
disease severity determined by the model of end-stage liver disease (MELD)
<span class="citation">(Malinchoc et al. 2000; Wiesner et al. 2003; Organ Procurement and Transplantation Network 2021)</span>.
The MELD score estimates the patients’ 3-month mortality risk based on
laboratory results, namely bilirubin, creatinine and international normalized
ratio (INR). In general the MELD score is extended by the sodium level
(MELD-Na score) as this was found to be an important additional risk factor in
liver cirrhosis <span class="citation">(Kim et al. 2008; Organ Procurement and Transplantation Network 2021)</span>.
The MELD was initially developed for predicting the survival of patients
undergoing transjugular intrahepatic portosystemic shunts.
Afterwards it was revalidated for predicting mortality risk in patients awaiting
a liver transplantation.
Although the MELD score should be an objective allocation score especially the
creatinine and INR are highly dependent on the utilized laboratory methods
<span class="citation">(Trotter et al. 2004; Cholongitas et al. 2007)</span>. Patients with identical disease state could
have very different MELD scores and thus get different priority on the liver
transplantation waiting list.
Furthermore often, e.g. for acute-on-chronic liver failure, the MELD score
underestimates the mortality risk <span class="citation">(Hernaez et al. 2020)</span>.</p>
<p>There have been some attempts to use the data extracted from more than 300.000
electronic medical records from two hospitals in the United States to improve
the MELD score <span class="citation">(Kartoun et al. 2017)</span>. The derived MELD-Plus7 and MELD-Plus9 risk score
add albumin, white blood cell count, total cholesterol, age and length of
stay to the MELD-Na variables. Despite its published prediction improvement the
MELD-Plus scores are not used for transplant allocation yet.</p>
<p>As depicted by MELD-Plus risk scores better predictive scores often need more
variables and are more complicated. To reduce the risk of overlooking or
incorrectly calculating and interpreting the results,
clinical decision support systems may be used and could improve patient safety.
The research project on digital laboratory medicine (AMPEL) develops a
clinical decision support system based on laboratory diagnostics that should
support clinical practitioners in interpreting the laboratory results and taking
the necessary medical interventions <span class="citation">(Eckelt et al. 2020)</span>.</p>
<p>This study aims to find clinical and laboratory values that improve the risk
stratification for liver transplantation over classical MELD, MELD-Na and
MELD-Plus scores and could be implemented as part of the AMPEL clinical decision
support system.</p>
</div>
<div id="material-and-methods" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Material and methods</h1>
<div id="study-population" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Study population</h2>
<p>In a retrospective cohort study we followed 778 consecutive
patients, who were recruited during the evaluation process for
liver transplantation at the University Hospital of Leipzig from
November 2012 to June 2015.
For each patient we recorded 44 variables.
Among them are age, sex, etiology of liver disease, complications as listed in
Table <a href="#tab:tbl1">3.1</a> and 25 laboratory measurements.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:flowchart"></span>
<div id="htmlwidget-6cd908ff8883cd327a53" style="width:480px;height:480px;" class="grViz html-widget"></div>
<script type="application/json" data-for="htmlwidget-6cd908ff8883cd327a53">{"x":{"diagram":"\ndigraph flowchart {\n    graph [splines = false];\n\n    // invisible nodes\n    node [shape = point, width = 0, height = 0];\n    iAR [style = \"invis\"];\n\n    // visible nodes\n    node [shape = box];\n    A [label = \"Total Records\n(n = 778)\"];\n    E [label = \"Excluded (n = 124)\\l- Younger than 18 years: 1\\l- Liver Transplantation before visit: 24\\l- Follow-up missing/invalid: 99\\l\"];\n    R [label = \"Analysed Records\n(n = 654)\"];\n\n    // edge statements\n    A -> iAR [arrowhead = none];\n    {rank = same; iAR -> E [minlen = 5]};\n    iAR -> R;\n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.1: Flowchart Inclusion/Exclusion.
</p>
</div>
<p>We excluded 124 patients from our analysis
(Figure <a href="#fig:flowchart">2.1</a>).
One was younger than 18 years.
24 had a liver transplantation before and
in 99 cases the follow up data were missing or
invalid.</p>
<p>The Ethics Committee at the Leipzig University Faculty of
Medicine approved the retrospective usage of the data for our study
(reference number: 039/14ff).</p>
</div>
<div id="meld-scores" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> MELD scores</h2>
<p>MELD and MELD-Na were calculated as described in
<span class="citation">Organ Procurement and Transplantation Network (2021)</span> using the following formulas:
<span class="math inline">\(MELD = 10 * (0.957 \ln(creatinine [mg/dl]) + 0.378 \ln(bilirubin [mg/dl]) + 1.120 * \ln(INR) + 0.643)\)</span>,
Creatinine, bilirubin and INR values lower than 1.0 mg/dl were set to 1.0 mg/dl.
The maximum allowed creatinine was 4.0 mg/dl. If patients received dialysis,
creatinine was set to 4.0 mg/dl.
For MELD-Na MELD was calculated as above and recalculated if greater than 11 using:
<span class="math inline">\(MELD\text{-}Na = MELD_i + 1.32 * (137 - Na [mmol/l]) - (0.033 * MELD_i * (137 - Na [mmol/l]))\)</span></p>
<p>Serum sodium values lower than 125 mmol/l or higher than 137 mmol/l were set
to 125 mmol/l and 137 mmol/l, respectively.</p>
<p>The MELD-Plus7 risk score was calculated as described in <span class="citation">Kartoun et al. (2017)</span>:
<span class="math display">\[
\begin{aligned}
L = 8.53499496 &amp;+ \\
    2.59679650 &amp;* \log_{10}(1 + creatinine [mg/dl]) + \\
    2.06503238 &amp;* \log_{10}(1 + bilirubin [mg/dl]) + \\
    2.99724802 &amp;* \log_{10}(1 + INR) - \\
    6.47834101 &amp;* \log_{10}(1 + sodium [mmol/l]) - \\
    6.34990436 &amp;* \log_{10}(1 + albumin [g/l]) + \\
    1.92811726 &amp;* \log_{10}(1 + wbc [th/cumm]) + \\
    0.04070442 &amp;* age [years]
\end{aligned}
\]</span></p>
<p><span class="math display">\[MELD\text{-}Plus = \frac{\exp(L)}{1 + \exp(L)}\]</span></p>
<p>The MELD-Plus9 score was ignored because the length of stay information was not
available for our cohort and the model itself wasn’t superior to MELD-Plus7 in
its original publication <span class="citation">(Kartoun et al. 2017)</span>.</p>
</div>
<div id="data-processing-and-machine-learning" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Data processing and machine learning</h2>
<p>All data processing, statistical and machine learning analyses were performed
using R version
4.2.0
<span class="citation">(R Core Team 2022)</span>.
Prior to the analyses all laboratory values were <em>zlog</em> transformed
as described in <span class="citation">Hoffmann et al. (2017)</span> and implemented in <span class="citation">Gibb (2021)</span>.</p>
<p><span class="math display">\[
z = (\log(x) - \frac{\log(LL) + \log(UL)}{2}) * \frac{3.92}{\log(UL) - \log(LL)}
\]</span></p>
<p>Where <span class="math inline">\(LL\)</span> and <span class="math inline">\(UL\)</span> are the lower and upper reference limit of
the corresponding laboratory diagnostic, respectively.
Missing laboratory measurements were replaced with a <em>zlog</em> value of zero.
In similar manner if information about complications or
comorbidities were missing we treat them as not present.</p>
<p>We compared 13 different statistical and machine learning
algorithms using the <em>mlr3</em> framework
<span class="citation">(Lang et al. 2019, 2022; Sonabend et al. 2021; Sonabend, Kiraly, and Lang 2022)</span>.
The used algorithms are designed for analysis of survival data:
Cox proportional hazards regression model
<span class="citation">(Cox 1972; Terry M. Therneau and Patricia M. Grambsch 2000; Therneau 2022)</span>,
penalized regressions,
namely two different implementations of the
lasso (least absolute shrinkage and selection operator) regression
<span class="citation">(Tibshirani 1997; Simon et al. 2011; Friedman et al. 2022; J. J. Goeman 2010; J. Goeman et al. 2022)</span>,
two different implementations of the
ridge regression <span class="citation">(Simon et al. 2011; Friedman et al. 2022; J. J. Goeman 2010; J. Goeman et al. 2022)</span>, and
the elastic net regression <span class="citation">(Simon et al. 2011; Friedman et al. 2022)</span>,
two different random forest implementations
<span class="citation">(Wright and Ziegler 2017; Wright, Wager, and Probst 2021; H. Ishwaran et al. 2008; Hemant Ishwaran and Kogalur 2020)</span>,
extreme gradient boosting <span class="citation">(T. Chen and Guestrin 2016; Tianqi Chen et al. 2022)</span>,
two different support vector machines <span class="citation">(Belle et al. 2011; Fouodo 2018)</span>
and two neural networks <span class="citation">(Katzman et al. 2018; Kvamme, Borgan, and Scheel 2019; Sonabend 2022)</span>.</p>
<p>To choose the best model we used nested resampling to avoid feature- and
model-selection-bias.
Therefore, we applied a 5-fold inner
and a 3-fold outer cross-validation
that was repeated 10 times.
We searched the hyperparameter space using a grid search with a resolution of
10.
Subsequently we selected the model with the
highest median concordance index <span class="citation">(Harrell 1982)</span>. If multiple models were
comparable we chose the simpler (more penalized) one.</p>
<p>After selecting the final model we trained and validated it as described in
<span class="citation">Harrell, Lee, and Mark (1996)</span>.
Briefly we developed our model using a 3 cross-validation
that was repeated 10 times and finally predicted survival
at 90 days.
The predictions were grouped into equal sized intervals so that there
are 50 patients in each interval.
For each group the mean survival probability and the difference to the
Kaplan-Meier estimate was calculated.
The same procedure was repeated for 200 bootstrap samples.
Subsequently the differences between predicted survival and Kaplan-Meier
estimates of the bootstrap samples were averaged and added to the original
model difference to obtain a bias-corrected estimated between predicted and
observed survival.</p>
<p>The final model was compared against the MELD, MELD-Na and MELD-Plus7 score
using the area under the time-dependent receiver operating characteristic curve
(AUROC) based on the nonparametric inverse probability of censoring weighting
estimator (IPCW) <span class="citation">Paul Blanche (2019)</span>.
Testing was applied as described in <span class="citation">P. Blanche, Dartigues, and Jacqmin-Gadda (2013)</span>.
A <span class="math inline">\(p &lt; 0.05\)</span> was considered as statistical significant difference.</p>
<p>Summary tables were built using the <em>gtsummary</em> package
<span class="citation">(Sjoberg et al. 2021, 2022)</span>.</p>
</div>
</div>
<div id="results" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Results</h1>
<div id="baseline-characteristics" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Baseline characteristics</h2>
<div id="ziidnstgxp" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

#ziidnstgxp .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 13px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ziidnstgxp .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ziidnstgxp .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ziidnstgxp .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ziidnstgxp .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ziidnstgxp .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ziidnstgxp .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ziidnstgxp .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ziidnstgxp .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ziidnstgxp .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ziidnstgxp .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ziidnstgxp .gt_group_heading {
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
}

#ziidnstgxp .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ziidnstgxp .gt_from_md > :first-child {
  margin-top: 0;
}

#ziidnstgxp .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ziidnstgxp .gt_row {
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ziidnstgxp .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ziidnstgxp .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ziidnstgxp .gt_row_group_first td {
  border-top-width: 2px;
}

#ziidnstgxp .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
}

#ziidnstgxp .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#ziidnstgxp .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ziidnstgxp .gt_last_summary_row {
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ziidnstgxp .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
}

#ziidnstgxp .gt_first_grand_summary_row {
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ziidnstgxp .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ziidnstgxp .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ziidnstgxp .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ziidnstgxp .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-left: 1px;
  padding-right: 1px;
  padding-left: 5px;
  padding-right: 5px;
}

#ziidnstgxp .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ziidnstgxp .gt_sourcenote {
  font-size: 90%;
  padding-top: 1px;
  padding-bottom: 1px;
  padding-left: 5px;
  padding-right: 5px;
}

#ziidnstgxp .gt_left {
  text-align: left;
}

#ziidnstgxp .gt_center {
  text-align: center;
}

#ziidnstgxp .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ziidnstgxp .gt_font_normal {
  font-weight: normal;
}

#ziidnstgxp .gt_font_bold {
  font-weight: bold;
}

#ziidnstgxp .gt_font_italic {
  font-style: italic;
}

#ziidnstgxp .gt_super {
  font-size: 65%;
}

#ziidnstgxp .gt_two_val_uncert {
  display: inline-block;
  line-height: 1em;
  text-align: right;
  font-size: 60%;
  vertical-align: -0.25em;
  margin-left: 0.1em;
}

#ziidnstgxp .gt_footnote_marks {
  font-style: italic;
  font-weight: normal;
  font-size: 75%;
  vertical-align: 0.4em;
}

#ziidnstgxp .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ziidnstgxp .gt_slash_mark {
  font-size: 0.7em;
  line-height: 0.7em;
  vertical-align: 0.15em;
}

#ziidnstgxp .gt_fraction_numerator {
  font-size: 0.6em;
  line-height: 0.6em;
  vertical-align: 0.45em;
}

#ziidnstgxp .gt_fraction_denominator {
  font-size: 0.6em;
  line-height: 0.6em;
  vertical-align: -0.05em;
}
</style>
<table class="gt_table">
  <caption><span id="tab:tbl1">Table 3.1: </span>Baseline characteristics.<br />Values are given as median (lower quartile, upper quartile) or n (percent).<br />AIH, autoimmune hepatitis; ALAT, alanine aminotransferase; ALF, acute liver failure; ASAT, aspartate aminotransferase; CLI, chronic liver insufficiency; GIB, gastrointestinal bleeding; HBV, hepatitis B virus; HCV, hepatitis C virus; IL-6, interleukin 6; INR, international normalized ratio; LTx, liver transplantation; NASH, non-alcoholic steatohepatitis; PBC, primary biliary cirrhosis; PSC, primary sclerosing cholangitis; SBP, spontaneous bacterial peritonitis</caption>
  
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1"><strong>Characteristic</strong></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1"><strong>Female</strong>, N = 240</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1"><strong>Male</strong>, N = 414</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1"><strong>Overall</strong>, N = 654</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">General</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Follow-up time [days]</td>
<td class="gt_row gt_center">181 (17, 411)</td>
<td class="gt_row gt_center">198 (54, 378)</td>
<td class="gt_row gt_center">191 (38, 384)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Age</td>
<td class="gt_row gt_center">56 (51, 64)</td>
<td class="gt_row gt_center">58 (52, 63)</td>
<td class="gt_row gt_center">58 (52, 64)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">LTx</td>
<td class="gt_row gt_center">20 (8.3%)</td>
<td class="gt_row gt_center">40 (9.7%)</td>
<td class="gt_row gt_center">60 (9.2%)</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">Laboratory measurements</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Total bilirubin [µmol/l]</td>
<td class="gt_row gt_center">21 (10, 49)</td>
<td class="gt_row gt_center">26 (14, 51)</td>
<td class="gt_row gt_center">24 (13, 50)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Cystatin C [mg/l]</td>
<td class="gt_row gt_center">1.31 (1.04, 1.80)</td>
<td class="gt_row gt_center">1.32 (1.08, 1.76)</td>
<td class="gt_row gt_center">1.32 (1.07, 1.79)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">5</td>
<td class="gt_row gt_center">7</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Creatinine [µmol/l]</td>
<td class="gt_row gt_center">73 (58, 91)</td>
<td class="gt_row gt_center">85 (71, 110)</td>
<td class="gt_row gt_center">80 (65, 104)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">INR</td>
<td class="gt_row gt_center">1.20 (1.06, 1.50)</td>
<td class="gt_row gt_center">1.22 (1.10, 1.44)</td>
<td class="gt_row gt_center">1.21 (1.09, 1.46)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">2</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Sodium [mmol/l]</td>
<td class="gt_row gt_center">138.8 (135.4, 141.0)</td>
<td class="gt_row gt_center">138.0 (135.0, 140.1)</td>
<td class="gt_row gt_center">138.1 (135.2, 140.4)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">5</td>
<td class="gt_row gt_center">4</td>
<td class="gt_row gt_center">9</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">WBC [exp 9/l]</td>
<td class="gt_row gt_center">6.7 (4.7, 8.8)</td>
<td class="gt_row gt_center">6.2 (4.6, 7.8)</td>
<td class="gt_row gt_center">6.3 (4.7, 8.3)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">10</td>
<td class="gt_row gt_center">13</td>
<td class="gt_row gt_center">23</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">IL-6 [pg/ml]</td>
<td class="gt_row gt_center">10 (5, 36)</td>
<td class="gt_row gt_center">14 (6, 47)</td>
<td class="gt_row gt_center">13 (6, 42)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">27</td>
<td class="gt_row gt_center">29</td>
<td class="gt_row gt_center">56</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Albumin [g/l]</td>
<td class="gt_row gt_center">41 (35, 46)</td>
<td class="gt_row gt_center">38 (33, 43)</td>
<td class="gt_row gt_center">39 (34, 45)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">4</td>
<td class="gt_row gt_center">6</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Total protein [g/l]</td>
<td class="gt_row gt_center">71 (63, 76)</td>
<td class="gt_row gt_center">71 (65, 76)</td>
<td class="gt_row gt_center">71 (65, 76)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">4</td>
<td class="gt_row gt_center">6</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Cholesterine [mmol/l]</td>
<td class="gt_row gt_center">4.64 (3.41, 5.56)</td>
<td class="gt_row gt_center">4.28 (3.34, 5.15)</td>
<td class="gt_row gt_center">4.40 (3.34, 5.31)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">ALAT [µkat/l]</td>
<td class="gt_row gt_center">0.29 (0.21, 0.42)</td>
<td class="gt_row gt_center">0.36 (0.23, 0.56)</td>
<td class="gt_row gt_center">0.33 (0.22, 0.48)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">8</td>
<td class="gt_row gt_center">10</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">ASAT [µkat/l]</td>
<td class="gt_row gt_center">0.68 (0.49, 1.05)</td>
<td class="gt_row gt_center">0.81 (0.61, 1.23)</td>
<td class="gt_row gt_center">0.77 (0.55, 1.16)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">3</td>
<td class="gt_row gt_center">5</td>
<td class="gt_row gt_center">8</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">MELD</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">MELD score</td>
<td class="gt_row gt_center">10 (7, 18)</td>
<td class="gt_row gt_center">12 (9, 17)</td>
<td class="gt_row gt_center">11 (8, 17)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">3</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">MELD Category</td>
<td class="gt_row gt_center"></td>
<td class="gt_row gt_center"></td>
<td class="gt_row gt_center"></td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">[6,9]</td>
<td class="gt_row gt_center">111 (47%)</td>
<td class="gt_row gt_center">146 (35%)</td>
<td class="gt_row gt_center">257 (39%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">[10,20)</td>
<td class="gt_row gt_center">80 (34%)</td>
<td class="gt_row gt_center">196 (47%)</td>
<td class="gt_row gt_center">276 (42%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">[20,30)</td>
<td class="gt_row gt_center">30 (13%)</td>
<td class="gt_row gt_center">50 (12%)</td>
<td class="gt_row gt_center">80 (12%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">[30,40)</td>
<td class="gt_row gt_center">12 (5.0%)</td>
<td class="gt_row gt_center">14 (3.4%)</td>
<td class="gt_row gt_center">26 (4.0%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">[40,52)</td>
<td class="gt_row gt_center">5 (2.1%)</td>
<td class="gt_row gt_center">7 (1.7%)</td>
<td class="gt_row gt_center">12 (1.8%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">3</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">MELD-Na score</td>
<td class="gt_row gt_center">10 (7, 19)</td>
<td class="gt_row gt_center">12 (9, 20)</td>
<td class="gt_row gt_center">11 (8, 20)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">5</td>
<td class="gt_row gt_center">4</td>
<td class="gt_row gt_center">9</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">Entities</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Cirrhosis</td>
<td class="gt_row gt_center">203 (85%)</td>
<td class="gt_row gt_center">391 (94%)</td>
<td class="gt_row gt_center">594 (91%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">1</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">ALF</td>
<td class="gt_row gt_center">6 (2.5%)</td>
<td class="gt_row gt_center">2 (0.5%)</td>
<td class="gt_row gt_center">8 (1.2%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">CLI</td>
<td class="gt_row gt_center">30 (12%)</td>
<td class="gt_row gt_center">21 (5.1%)</td>
<td class="gt_row gt_center">51 (7.8%)</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">Etiology (more than 1 per patient possible)</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Ethyltoxic</td>
<td class="gt_row gt_center">114 (48%)</td>
<td class="gt_row gt_center">297 (72%)</td>
<td class="gt_row gt_center">411 (63%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">HBV</td>
<td class="gt_row gt_center">4 (1.7%)</td>
<td class="gt_row gt_center">16 (3.9%)</td>
<td class="gt_row gt_center">20 (3.1%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">HCV</td>
<td class="gt_row gt_center">17 (7.1%)</td>
<td class="gt_row gt_center">28 (6.8%)</td>
<td class="gt_row gt_center">45 (6.9%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">AIH</td>
<td class="gt_row gt_center">22 (9.2%)</td>
<td class="gt_row gt_center">9 (2.2%)</td>
<td class="gt_row gt_center">31 (4.7%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">PBC</td>
<td class="gt_row gt_center">17 (7.1%)</td>
<td class="gt_row gt_center">0 (0%)</td>
<td class="gt_row gt_center">17 (2.6%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">PSC</td>
<td class="gt_row gt_center">5 (2.1%)</td>
<td class="gt_row gt_center">11 (2.7%)</td>
<td class="gt_row gt_center">16 (2.4%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">NASH</td>
<td class="gt_row gt_center">17 (7.1%)</td>
<td class="gt_row gt_center">31 (7.5%)</td>
<td class="gt_row gt_center">48 (7.3%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Cryptogenic</td>
<td class="gt_row gt_center">36 (15%)</td>
<td class="gt_row gt_center">32 (7.7%)</td>
<td class="gt_row gt_center">68 (10%)</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">Complications (more than 1 per patient possible)</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Dialysis</td>
<td class="gt_row gt_center">14 (5.9%)</td>
<td class="gt_row gt_center">19 (4.6%)</td>
<td class="gt_row gt_center">33 (5.1%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">2</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">GIB</td>
<td class="gt_row gt_center">58 (24%)</td>
<td class="gt_row gt_center">104 (25%)</td>
<td class="gt_row gt_center">162 (25%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">HCC</td>
<td class="gt_row gt_center">18 (7.5%)</td>
<td class="gt_row gt_center">103 (25%)</td>
<td class="gt_row gt_center">121 (19%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">SBP</td>
<td class="gt_row gt_center">31 (13%)</td>
<td class="gt_row gt_center">60 (15%)</td>
<td class="gt_row gt_center">91 (14%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 20px;">(Missing)</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">1</td></tr>
    <tr class="gt_group_heading_row">
      <td colspan="4" class="gt_group_heading">Mortality</td>
    </tr>
    <tr class="gt_row_group_first"><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Within 7 days</td>
<td class="gt_row gt_center">16 (7.2%)</td>
<td class="gt_row gt_center">18 (4.5%)</td>
<td class="gt_row gt_center">34 (5.5%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Within 30 days</td>
<td class="gt_row gt_center">25 (12%)</td>
<td class="gt_row gt_center">31 (8.1%)</td>
<td class="gt_row gt_center">56 (9.4%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Within 90 days</td>
<td class="gt_row gt_center">32 (16%)</td>
<td class="gt_row gt_center">49 (13%)</td>
<td class="gt_row gt_center">81 (14%)</td></tr>
    <tr><td class="gt_row gt_left" style="text-align: left; text-indent: 10px;">Within 365 days</td>
<td class="gt_row gt_center">39 (20%)</td>
<td class="gt_row gt_center">61 (18%)</td>
<td class="gt_row gt_center">100 (18%)</td></tr>
  </tbody>
  
  
</table>
</div>
<p>In this study we analysed a cohort of 654 patients evaluated for
liver transplantation (Tab. <a href="#tab:tbl1">3.1</a>).
414 (63%)
of them were men.
The median (IQR) follow-up times was 191 (38, 384) days.
Within the follow-up time 60 (9.2%) patients received a liver transplant
and were censored for the analysis beyond this time point.
The median (IQR) age of the cohort was 58 (52, 64) years.</p>
<p>Most of the patients presented with cirrhosis
(594 (91%)).
The remaining patients had an acute liver failure
(8 (1.2%)) or
a chronic liver insufficiency
(51 (7.8%)).
At evaluation time the median (IQR) MELD and MELD-Na score were
11 (8, 17) and
11 (8, 20), respectively.
Within 90 days 81 (14%) patients died.
In nearly two third of all patients alcohol was the main cause
(411 (63%)) of their end-stage liver disease.
The remaining patients had viral infections
(hepatitis B and C viruse 20 (3.1%) and 45 (6.9%), respectively),
autoimmune processes (autoimmune hepatitis 31 (4.7%),
primary sclerosing cirrhosis 16 (2.4%)),
primary biliary cirrhosis 17 (2.6%) or unknown causes
(non-alcoholic steatohepatitis 48 (7.3%) and
cryptogenic hepatitis 68 (10%)).
The most common complications among all patients were gastrointestinal
bleedings in 162 (25%) patients, followed by
hepatocellular carcinoma (HCC) in 121 (19%) and
spontaneous bacterial peritonitis (SBP) in 91 (14%) patients.
Just 33 (5.1%) patients required a renal dialysis.</p>
</div>
<div id="meld-scores-1" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> MELD scores</h2>
<p>According to <span class="citation">Wiesner et al. (2003)</span> we divided our cohort into
5 MELD-score risk
categories, with 257, 276, 80, 26, and 12 patients,
respectively.
As shown in Table <a href="#tab:tbl-observed-vs-expected-mortality">3.2</a>
our observed 90-day mortality was
1.4 to 2.9
times higher than the predicted one. Just in the group with the lowest MELD
scores the mortality was much lower.</p>
<table>
<caption><span id="tab:tbl-observed-vs-expected-mortality">Table 3.2: </span>Observed vs MELD-expected 90-day mortality. MELD mortality values are taken from <span class="citation">Wiesner et al. (2003)</span>. All patients censored before day 90 are ignored for the calculation of the MELD-expected deaths. SMR, Standardized mortality ratio = observed deaths/expected deaths.</caption>
<colgroup>
<col width="10%" />
<col width="14%" />
<col width="14%" />
<col width="25%" />
<col width="17%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">MELD category</th>
<th align="right">Observed deaths (n)</th>
<th align="right">Expected deaths (n)</th>
<th align="right">Standardized mortality ratio (SMR)</th>
<th align="right">Observed mortality (%)</th>
<th align="right">Expected mortality (%)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">[6,9]</td>
<td align="right">1</td>
<td align="right">4.0</td>
<td align="right">0.3</td>
<td align="right">0.5</td>
<td align="right">1.9</td>
</tr>
<tr class="even">
<td align="left">[10,20)</td>
<td align="right">21</td>
<td align="right">13.0</td>
<td align="right">1.6</td>
<td align="right">8.9</td>
<td align="right">6.0</td>
</tr>
<tr class="odd">
<td align="left">[20,30)</td>
<td align="right">34</td>
<td align="right">11.6</td>
<td align="right">2.9</td>
<td align="right">51.0</td>
<td align="right">19.6</td>
</tr>
<tr class="even">
<td align="left">[30,40)</td>
<td align="right">18</td>
<td align="right">10.0</td>
<td align="right">1.8</td>
<td align="right">92.8</td>
<td align="right">52.6</td>
</tr>
<tr class="odd">
<td align="left">[40,52)</td>
<td align="right">6</td>
<td align="right">4.3</td>
<td align="right">1.4</td>
<td align="right">100.0</td>
<td align="right">71.3</td>
</tr>
</tbody>
</table>
</div>
<div id="ampel-meld" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> AMPEL MELD</h2>
<p>The penalized regression and random forest models
result in a similar high median performance in the benchmark process.
In favour of simpler models we chose a lasso based approach for the final model
development.
The classical cox model, the data and computational intense algorithms
extreme gradient boosting, support vector machines and neural networks yield a
worse performance.</p>
<table>
<caption><span id="tab:coeftbl">Table 3.3: </span>Model coefficients</caption>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Coefficient</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">IL-6</td>
<td align="right">0.193</td>
</tr>
<tr class="even">
<td align="left">Total protein</td>
<td align="right">-0.081</td>
</tr>
<tr class="odd">
<td align="left">INR</td>
<td align="right">0.067</td>
</tr>
<tr class="even">
<td align="left">Cholinesterase</td>
<td align="right">-0.037</td>
</tr>
<tr class="odd">
<td align="left">Cystatin C</td>
<td align="right">0.026</td>
</tr>
<tr class="even">
<td align="left">Direct bilirubin</td>
<td align="right">0.010</td>
</tr>
</tbody>
</table>
<p>The resulting model AMPEL model of end-stage liver disease
(AMELD) uses 6 predictors.
Beside the classical MELD variables INR and (direct) bilirubin the lasso
regression selected cystatin c over creatinine, and
additionally IL-6, total protein and cholinesterase.</p>
<table>
<caption><span id="tab:basehaz">Table 3.4: </span>Underlying survival function; S<sub>0</sub>: baseline survival estimate</caption>
<thead>
<tr class="header">
<th align="left">t [days]</th>
<th align="right">2</th>
<th align="right">7</th>
<th align="right">30</th>
<th align="right">90</th>
<th align="right">180</th>
<th align="right">365</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">S<sub>0</sub> (t)</td>
<td align="right">0.0033</td>
<td align="right">0.0088</td>
<td align="right">0.0184</td>
<td align="right">0.0339</td>
<td align="right">0.0454</td>
<td align="right">0.0511</td>
</tr>
</tbody>
</table>
<p>The 90-day survival estimate of a given patient could be calculated as shown in
<a href="#eq:survcalc">(3.1)</a>.</p>
<p><span class="math display" id="eq:survcalc">\[\begin{align} S &amp;= S_{0,90}^{\exp{LP}} \\ &amp;=  0.0339 ^{\exp{( 0.1929 * zlog(\text{IL-6})-0.0805 * zlog(\text{Total protein})+0.0667 * zlog(\text{INR})-0.0368 * zlog(\text{Cholinesterase})+0.026 * zlog(\text{Cystatin C})+0.0098 * zlog(\text{Direct bilirubin}) )}}
\tag{3.1}
\end{align}
\]</span>
<!-- using eq:survcalc inside the R code chunk impedes Math ML rendering --></p>
<p>Where <span class="math inline">\(S_{0}\)</span> is the baseline survival estimate from
<a href="#tab:basehaz">3.4</a> and <span class="math inline">\(LP\)</span> is the linear predictor of the coefficients
in <a href="#tab:coeftbl">3.3</a>.</p>
</div>
<div id="model-comparison" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> Model comparison</h2>
<p><img src="figure/article.Rmd/roc-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-roc-1">
Past versions of roc-1.png
</button>
</p>
<div id="fig-roc-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/ebe29cf9f65cea961c6ea141aa04514818137f8c/docs/figure/article.Rmd/roc-1.png" target="_blank">ebe29cf</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/803521940699fbefb650f4567d9d5872332097d1/docs/figure/article.Rmd/roc-1.png" target="_blank">8035219</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<table>
<caption><span id="tab:roctest">Table 3.5: </span>Test comparison between AUCROC for MELD-Na and AMELDaccording to <span class="citation">P. Blanche, Dartigues, and Jacqmin-Gadda (2013)</span> at day 2, 7, 30, 90, 180, 365.<span class="math inline">\(p\)</span>-values are shown as non-adjusted and adjusted for multiple testing.</caption>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">t=2</th>
<th align="right">t=7</th>
<th align="right">t=30</th>
<th align="right">t=90</th>
<th align="right">t=180</th>
<th align="right">t=365</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Non-adjusted <span class="math inline">\(p\)</span>-values</td>
<td align="right">0.27</td>
<td align="right">0.15</td>
<td align="right">0.33</td>
<td align="right">0.03</td>
<td align="right">0.02</td>
<td align="right">0.33</td>
</tr>
<tr class="even">
<td align="left">Adjusted <span class="math inline">\(p\)</span>-values</td>
<td align="right">0.75</td>
<td align="right">0.52</td>
<td align="right">0.84</td>
<td align="right">0.15</td>
<td align="right">0.08</td>
<td align="right">0.84</td>
</tr>
</tbody>
</table>
<p>AMELD offers the best AUROC (0.961) across all competitors.
However, the difference against the second best model MELD-Na
(AUROC: 0.936) is not statistical significant (
<span class="math inline">\(p = 0.15\)</span>) after correction for multiple
testing (see Table <a href="#tab:roctest">3.5</a> for complete comparison).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:bootrcv"></span>
<img src="figure/article.Rmd/bootrcv-1.png" alt="Calibration of AMELD in comparison to MELD, MELD-Na and MELD-Plus7." width="768" />
<p class="caption">
Figure 3.1: Calibration of AMELD in comparison to MELD, MELD-Na and MELD-Plus7.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-bootrcv-1">
Past versions of bootrcv-1.png
</button>
</p>
<div id="fig-bootrcv-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/ebe29cf9f65cea961c6ea141aa04514818137f8c/docs/figure/article.Rmd/bootrcv-1.png" target="_blank">ebe29cf</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/ampel-leipzig/ampel-leipzig-meld/blob/803521940699fbefb650f4567d9d5872332097d1/docs/figure/article.Rmd/bootrcv-1.png" target="_blank">8035219</a>
</td>
<td>
Sebastian Gibb
</td>
<td>
2022-06-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Fig. <a href="#fig:bootrcv">3.1</a> depicts the calibration curve for AMELD and the
competing MELD scores.
While the prediction for low-risk patients is similar to other scores
AMELD predicts the medium-risk patients more accurate.
Because of the low number of very ill patients in our cohort the prediction
performance in high-risk patients is poor but comparable to the other models.
This would require a recalibration in a larger validation cohort.</p>
</div>
</div>
<div id="discussion" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Discussion</h1>
<p>In this retrospective analysis of 654 consecutive patients,
who were recruited during the evaluation process for liver transplantation at
the University Hospital of Leipzig we developed a new model to predict the
90-day survival probability. Our new model, AMELD, outperforms the MELD-Na
score and the recently published MELD-Plus7 risk score.</p>
<p>Although the Organ Procurement and Transplantation Network recommends the use
of MELD-Na over MELD the sodium does not add any information to the score.
Neither in the bootstrap selection nor in the variable importance of a
random forest model the sodium gains any important weight
(in bootstrap selection rank 18 of 42 and in a random forest variable importance rank 20 of 42, see <a href="#fig:vimpbs"><strong>??</strong></a>, <a href="#fig:vimprf"><strong>??</strong></a>).
Already <span class="citation">Kartoun et al. (2017)</span> have shown that the
discrimination performance of MELD-Na is insignificantly different from the
classical MELD.
Consequentially it is not a component of our model.</p>
<p>It has shown that women get lower scores in MELD-Na and therefore
are less likely getting an urgent listing for liver transplantation
<span class="citation">(<strong>gaertner2022?</strong>)</span>.
To address sexual differences in laboratory measurements we applied the <em>zlog</em>
transformation <span class="citation">(Hoffmann et al. 2017)</span>.
Additionally this transformation yield comparable laboratory measurements
across different clinical laboratories and methods.
It also reduce the influence of age- and sex-related differences,
which could improve the objectivity of the allocation process.
The already nearly normal scaled <em>zlog</em>-transformed laboratory measurements
allow us to avoid the sample depending scaling prior to model development.
Both should increase the external applicability of our approach.</p>
<p>Our model, AMELD, uses 6 predictors.
Half of them, INR, (direct) bilirubin, and cystatin c are identical
or similar to the labortory measurements utilized in the classical MELD.
The direct bilirubin selected by our approach and the total bilirubin in MELD
are highly correlated (spearman correlation:
1) and
therefore interchangeable.</p>
<p>In our feature selection process cystatin C was chosen instead of creatinine.
This is inline with recent studies that show a better prediction of mortality
for cystatin C in unselected and critical ill patients
<span class="citation">(Helmersson-Karlqvist, Ärnlöv, and Larsson 2016; Helmersson-Karlqvist et al. 2021)</span></p>
<p>Furthermore AMELD extends MELD by additional synthesis parameters
total protein and cholinesterase and
most importantly the inflammatory marker IL-6.</p>
<p>The latter is an example where results from machine learning can aid in
finding clinical meaningful patterns and guide further research.
The systemic inflammatory response syndrome is an
independent risk factor for poor outcome in patients with cirrhosis
<span class="citation">(Thabut et al. 2007)</span>.
However neither MELD nor MELD-Na taking inflammation into account.
In contrast IL-6 is the most important variable in our model. Even if IL-6 is
excluded the algorithms select CRP as one of the most important variables which
highlights the important role of inflammation in the prognosis of patients
with end-stage liver disease.
We and other researchers have already shown that adding inflammatory markers,
namely IL-6 or CRP, to the classical MELD score increase the accuracy of the
90-day mortality prediction <span class="citation">(Remmler et al. 2017; Cervoni et al. 2016)</span>.</p>
<p>Beside laboratory measurements we investigated
17
clinical data but none of them were chosen in our model selection process.
<span class="citation">Kartoun et al. (2017)</span> also used the lasso approach and neither select any clinical
data. Especially in retrospective analysis the clinical data are often not
detailed enough to keep valuable information.</p>
<p>In our benchmark of 13 different machine learning algorithms we have
seen that the more complex algorithm like boosting, support vector machines
and neural networks perform worse than simpler ones.
This may due additive effects that favor regression models or due to our
relative small sample size.
It has been shown that in machine learning more than 200 events per
predictor variable are required to achieve good prediction performance
<span class="citation">(Ploeg, Austin, and Steyerberg 2014)</span>.</p>
<!--
As we shown above machine learning can aid in finding clinical meaningful
patterns.
-->
<!--

- higher mortality than predicted, compare @hernaez2020
- adding sodium doesn't add much/any discrimination information
    - as already seen in @kartoun2017
- @kartoun2017 uses > 60 variables mostly clinical stuff, just few laboratory values, none of the clinical stuff remains
- general low sample size thus simpler models/fewer predictors are preferred.
- comparison of different predictors of MELD-Plus7 (+MELD-Plus9) vs our features
    - 90 day mortality *post* discharge
    - poorer performance in own validation set (lower and more uniform disease severity, also lower MELD)
- machine learning algorithms worse than statistical learning (lower c-index, higher variance/instability)
    - additive effects (favour statistical models)
    - low sample size, for ML more than 200 events per predictor variable necessary [@ploeg2014]
- sex-differences and sample-dependencies (by scaling) addressed by *zlog*.
- more variables, harder to cheat
- national register required
-->
<div id="limitations" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Limitations</h2>
<p>Our study has several limitations.
Principally it is a single center, retrospective analysis and without an
external validation cohort.
The data were collected during the evaluation process for liver transplantation
at the University Hospital of Leipzig.
There was no controlled follow-up process. Instead the follow-up information
were recored at the next clinical admission.
Furthermore we excluded all patients who had already a liver transplantation
which could artificial lower the disease severity of the cohort.
Next most of our patients suffered from ethyltoxic liver cirrhosis
(63%) which may
limited the transfer to other etiologies.</p>
<p>With a median MELD score of 11 (8, 17) our
cohort was healthier than in previous studies that reported
a score of 14-15 <span class="citation">(Kim et al. 2008; Kartoun et al. 2017)</span>.
However the 90-day mortality rate of 81 (14%) was comparable to
the mortality rates seen in earlier studies that ranges from 6 to 30%
<span class="citation">(Malinchoc et al. 2000; Kim et al. 2008; Kartoun et al. 2017)</span>.
Although our cohort was healthier with a medium mortality rate the classical
MELD score underestimates the observed mortality
1.4 to 2.9
times.
Unfortunately our sample size is very low in the high risk patient category
which yield a poor but better than MELD based prediction performance of our
AMELD for very ill patients.</p>
</div>
<div id="conclusion" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Conclusion</h2>
<p>In conclusion we developed a new model to predict the 90-day survival of
patients during the evaluation process for liver transplantation.
Our model, AMELD, extends the classical MELD predictors, INR,
(direct) bilirubin, cystatin c (instead of creatinine) by
synthesis parameters total protein and cholinesterase and
most importantly the inflammatory marker IL-6.
Using these 6 laboratory measurments the AMELD
outperforms the established MELD-Na and the recently published
MELD-Plus7 risk scores.
For a wider adoption a prospective multi center study for model
recalibration and validation is needed.</p>
<!--
Mortality
[@malinchoc2000] ~ 30 %
[@kim2008] ~ 6-7 %
[@kartoun2017] ~ 16.3 % (90 d, post-discharge-mortality)

random time point for blood sampling

It was not possible to ensure that patients died of end-stage liver disease.

The variables we found and the model itself have to be validated and
recalibrated in a multi center, prospective trial.

Few patients with high MELD scores, poor calibration for high mortality


- comparable mortality?
- retrospective analysis
- single center
- unknown cause of death (death with vs of end-stage liver disease)
- bias caused by LTx censorship?
-->
</div>
</div>
<div id="data-and-software-availability" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Data and software availability</h1>
<p>The complete code and reproducible analysis could be found at
<a href="https://ampel-leipzig.github.io/ampel-leipzig-meld/" class="uri">https://ampel-leipzig.github.io/ampel-leipzig-meld/</a>.
Additionally all data are available in the <code>ameld</code> R package <span class="citation">(Gibb 2022)</span> at
<a href="https://github.com/ampel-leipzig/ameld" class="uri">https://github.com/ampel-leipzig/ameld</a>
<a href="https://doi.org/10.5281/zenodo.6666018"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.6666018.svg" alt="DOI" /></a>
for further investigations.</p>
</div>
<div id="acknowledgement" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Acknowledgement</h1>
<p>The study was supported by the AMPEL project (www.ampel.care).
This project is co-financed through public funds according to the budget
decided by the Saxon State Parliament under the
RL eHealthSax 2017/18 grant number 100331796.
The funder provided support in the form of
salaries for author SG
but did not have any additional role in the study design,
data collection and analysis, decision to publish, or preparation
of the manuscript.
We thank the whole AMPEL team for their support,
and Stefan Kemnitz for his technical support regarding the use of the
HPC cluster of the University Greifswald.</p>
<p>The calculations were conducted using the high performance computing (HPC)
cluster of the University Greifswald.
In 2018 the average carbon efficiency of the German power grid was
0.471 kgCO<sub>2</sub>eq/kWh <span class="citation">(Icha, Lauf, and Kuhs 2021)</span>.
A single run of the complete calculation pipeline took roughly
96 hours of computation on an
Intel(R) Xeon(R) E5-2650 v4 and
Intel(R) Xeon(R) Gold 6240 CPU (thermal design power: 105 and 150 W).
The total estimated emissions were at least
6.77 kgCO<sub>2</sub>eq
(ignoring development, test runs, etc.).
This is equivalent to 44 km driven by an average car
<span class="citation">(Allekotte et al. 2021)</span>.</p>
</div>
<div id="references" class="section level1" number="7">
<h1><span class="header-section-number">7</span> References</h1>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-allekotte2021" class="csl-entry">
Allekotte, M., F. Bergk, K. Biemann, C. Deregowski, W. Knörr, H.-J. Althaus, D. Sutter, and T. Bergmann. 2021. <span>“Ökologische Bewertung von Verkehrsarten.”</span> 3716581060. Umweltbundesamt. <a href="https://www.umweltbundesamt.de/sites/default/files/medien/479/publikationen/texte_156-2020_oekologische_bewertung_von_verkehrsarten_0.pdf">https://www.umweltbundesamt.de/sites/default/files/medien/479/publikationen/texte_156-2020_oekologische_bewertung_von_verkehrsarten_0.pdf</a>.
</div>
<div id="ref-vanbelle2011" class="csl-entry">
Belle, V. Van, K. Pelckmans, S. Van Huffel, and J. A. K. Suykens. 2011. <span>“Support Vector Methods for Survival Analysis: A Comparison Between Ranking and Regression Approaches.”</span> <em>Artif Intell Med</em> 53 (2): 107–18. <a href="https://doi.org/10.1016/j.artmed.2011.06.006">https://doi.org/10.1016/j.artmed.2011.06.006</a>.
</div>
<div id="ref-R-timeROC" class="csl-entry">
Blanche, Paul. 2019. <em>timeROC: Time-Dependent ROC Curve and AUC for Censored Survival Data</em>. <a href="https://CRAN.R-project.org/package=timeROC">https://CRAN.R-project.org/package=timeROC</a>.
</div>
<div id="ref-blanche2013" class="csl-entry">
Blanche, P., J.-F. Dartigues, and H. Jacqmin-Gadda. 2013. <span>“Estimating and Comparing Time-Dependent Areas Under Receiver Operating Characteristic Curves for Censored Event Times with Competing Risks.”</span> <em>Stat Med</em> 32 (30): 5381–97. <a href="https://doi.org/10.1002/sim.5958">https://doi.org/10.1002/sim.5958</a>.
</div>
<div id="ref-cervoni2016" class="csl-entry">
Cervoni, J.-P., À. Amorós, R. Bañares, J. L. Montero, G. Soriano, D. Weil, R. Moreau, M. Pavesi, T. Thévenot, and V. Di Martino. 2016. <span>“Prognostic Value of c-Reactive Protein in Cirrhosis.”</span> <em>Eur J Gastroenterol Hepatol</em> 28 (9): 1028–34. <a href="https://doi.org/10.1097/meg.0000000000000676">https://doi.org/10.1097/meg.0000000000000676</a>.
</div>
<div id="ref-chen2016" class="csl-entry">
Chen, T., and C. Guestrin. 2016. <span>“<span>XGBoost</span>: <span>A</span> <span>Scalable</span> <span>Tree</span> <span>Boosting</span> <span>System</span>.”</span> <em>Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</em>, August, 785–94. <a href="https://doi.org/10.1145/2939672.2939785">https://doi.org/10.1145/2939672.2939785</a>.
</div>
<div id="ref-R-xgboost" class="csl-entry">
Chen, Tianqi, Tong He, Michael Benesty, Vadim Khotilovich, Yuan Tang, Hyunsu Cho, Kailong Chen, et al. 2022. <em>Xgboost: Extreme Gradient Boosting</em>. <a href="https://github.com/dmlc/xgboost">https://github.com/dmlc/xgboost</a>.
</div>
<div id="ref-cholongitas2007" class="csl-entry">
Cholongitas, E., L. Marelli, A. Kerry, M. Senzolo, D. W. Goodier, D. Nair, M. Thomas, D. Patch, and A. K. Burroughs. 2007. <span>“Different Methods of Creatinine Measurement Significantly Affect <span>MELD</span> Scores.”</span> <em>Liver Transpl</em> 13 (4): 523–29. <a href="https://doi.org/10.1002/lt.20994">https://doi.org/10.1002/lt.20994</a>.
</div>
<div id="ref-cox1972" class="csl-entry">
Cox, D. R. 1972. <span>“Regression Models and Life-Tables.”</span> <em>J R Stat Soc, Series B</em> 34 (2): 187–202. <a href="https://doi.org/10.1111/j.2517-6161.1972.tb00899.x">https://doi.org/10.1111/j.2517-6161.1972.tb00899.x</a>.
</div>
<div id="ref-eckelt2020" class="csl-entry">
Eckelt, F., J. Remmler, T. Kister, M. Wernsdorfer, H. Richter, M. Federbusch, M. Adler, et al. 2020. <span>“Verbesserte Patientensicherheit Durch <span>“</span>Clinical Decision Support Systems<span>”</span> in Der Labormedizin.”</span> <em>Der Internist</em> 61 (5): 452–59. <a href="https://doi.org/10.1007/s00108-020-00775-3">https://doi.org/10.1007/s00108-020-00775-3</a>.
</div>
<div id="ref-R-survivalsvm" class="csl-entry">
Fouodo, Cesaire J. K. 2018. <em>Survivalsvm: Survival Support Vector Analysis</em>. <a href="https://github.com/imbs-hl/survivalsvm">https://github.com/imbs-hl/survivalsvm</a>.
</div>
<div id="ref-R-glmnet" class="csl-entry">
Friedman, Jerome, Trevor Hastie, Rob Tibshirani, Balasubramanian Narasimhan, Kenneth Tay, Noah Simon, and James Yang. 2022. <em>Glmnet: Lasso and Elastic-Net Regularized Generalized Linear Models</em>. <a href="https://CRAN.R-project.org/package=glmnet">https://CRAN.R-project.org/package=glmnet</a>.
</div>
<div id="ref-R-zlog" class="csl-entry">
Gibb, Sebastian. 2021. <em><span class="nocase">zlog</span>: Z(log) Transformation for Laboratory Measurements</em>. <a href="https://doi.org/10.5281/zenodo.4727117">https://doi.org/10.5281/zenodo.4727117</a>.
</div>
<div id="ref-R-ameld" class="csl-entry">
———. 2022. <em>Data and Model of End-Stage Liver Disease Used in the <span>AMPEL</span> Project</em>. <a href="https://doi.org/10.5281/zenodo.6666018">https://doi.org/10.5281/zenodo.6666018</a>.
</div>
<div id="ref-penalized2010" class="csl-entry">
Goeman, J. J. 2010. <span>“L1 Penalized Estimation in the Cox Proportional Hazards Model.”</span> <em>Biometrical Journal</em>, no. 52, 1: –14.
</div>
<div id="ref-R-penalized" class="csl-entry">
Goeman, Jelle, Rosa Meijer, Nimisha Chaturvedi, and Matthew Lueder. 2022. <em>Penalized: L1 (Lasso and Fused Lasso) and L2 (Ridge) Penalized Estimation in GLMs and in the Cox Model</em>. <a href="https://CRAN.R-project.org/package=penalized">https://CRAN.R-project.org/package=penalized</a>.
</div>
<div id="ref-harrell1982" class="csl-entry">
Harrell, F. E. 1982. <span>“Evaluating the Yield of Medical Tests.”</span> <em>JAMA</em> 247 (18): 2543. <a href="https://doi.org/10.1001/jama.1982.03320430047030">https://doi.org/10.1001/jama.1982.03320430047030</a>.
</div>
<div id="ref-harrell1996" class="csl-entry">
Harrell, F. E., K. L. Lee, and D. B. Mark. 1996. <span>“MULTIVARIABLE PROGNOSTIC MODELS: ISSUES IN DEVELOPING MODELS, EVALUATING ASSUMPTIONS AND ADEQUACY, AND MEASURING AND REDUCING ERRORS.”</span> <em>Stat Med</em> 15 (4): 361–87. <a href="https://doi.org/10.1002/(sici)1097-0258(19960229)15:4&lt;361::aid-sim168&gt;3.0.co;2-4">https://doi.org/10.1002/(sici)1097-0258(19960229)15:4&lt;361::aid-sim168&gt;3.0.co;2-4</a>.
</div>
<div id="ref-helmerssonkarlqvist2016" class="csl-entry">
Helmersson-Karlqvist, J., J. Ärnlöv, and A. Larsson. 2016. <span>“Cystatin c-Based Glomerular Filtration Rate Associates More Closely with Mortality Than Creatinine-Based or Combined Glomerular Filtration Rate Equations in Unselected Patients.”</span> <em>Eur J Prev Cardiol</em> 23 (15): 1649–57. <a href="https://doi.org/10.1177/2047487316642086">https://doi.org/10.1177/2047487316642086</a>.
</div>
<div id="ref-helmerssonkarlqvist2021" class="csl-entry">
Helmersson-Karlqvist, J., M. Lipcsey, J. Ärnlöv, M. Bell, B. Ravn, A. Dardashti, and A. Larsson. 2021. <span>“Cystatin c Predicts Long Term Mortality Better Than Creatinine in a Nationwide Study of Intensive Care Patients.”</span> <em>Sci Rep</em> 11 (1). <a href="https://doi.org/10.1038/s41598-021-85370-8">https://doi.org/10.1038/s41598-021-85370-8</a>.
</div>
<div id="ref-hernaez2020" class="csl-entry">
Hernaez, R., Y. Liu, J. R. Kramer, A. Rana, H. B. El-Serag, and F. Kanwal. 2020. <span>“Model for End-Stage Liver Disease-Sodium Underestimates 90-Day Mortality Risk in Patients with Acute-on-Chronic Liver Failure.”</span> <em>J Hepatol</em>, June. <a href="https://doi.org/10.1016/j.jhep.2020.06.005">https://doi.org/10.1016/j.jhep.2020.06.005</a>.
</div>
<div id="ref-hoffmann2017" class="csl-entry">
Hoffmann, G., F. Klawonn, R. Lichtinghagen, and M. Orth. 2017. <span>“Der Zlog-Wert Als Basis Für Die Standardisierung von Laborwerten.”</span> <em><span>LaboratoriumsMedizin</span></em> 41 (1): 23–32. <a href="https://doi.org/10.1515/labmed-2016-0087">https://doi.org/10.1515/labmed-2016-0087</a>.
</div>
<div id="ref-icha2021" class="csl-entry">
Icha, P., T. Lauf, and G. Kuhs. 2021. <span>“Entwicklung Der Spezifischen Kohlendioxid-Emissionen Des Deutschen Strommix in Den Jahren 1990 - 2020.”</span> 7. <em>Climate Change</em>. Vol. 45. Umweltbundesamt. <a href="https://www.umweltbundesamt.de/sites/default/files/medien/5750/publikationen/2021-05-26_cc-45-2021_strommix_2021_0.pdf">https://www.umweltbundesamt.de/sites/default/files/medien/5750/publikationen/2021-05-26_cc-45-2021_strommix_2021_0.pdf</a>.
</div>
<div id="ref-R-randomForestSRC" class="csl-entry">
Ishwaran, Hemant, and Udaya B. Kogalur. 2020. <em>randomForestSRC: Fast Unified Random Forests for Survival, Regression, and Classification (RF-SRC)</em>. <a href="https://CRAN.R-project.org/package=randomForestSRC">https://CRAN.R-project.org/package=randomForestSRC</a>.
</div>
<div id="ref-randomForestSRC2008" class="csl-entry">
Ishwaran, H., U. B. Kogalur, E. H. Blackstone, and M. S. Lauer. 2008. <span>“Random Survival Forests.”</span> <em>Ann. Appl. Statist.</em> 2 (3): 841–60. <a href="http://arXiv.org/abs/0811.1645v1">http://arXiv.org/abs/0811.1645v1</a>.
</div>
<div id="ref-kartoun2017" class="csl-entry">
Kartoun, U., K. E. Corey, T. G. Simon, H. Zheng, R. Aggarwal, K. Ng, and S. Y. Shaw. 2017. <span>“The <span>MELD</span>-Plus: A Generalizable Prediction Risk Score in Cirrhosis.”</span> <em><span>PLOS</span> <span>ONE</span></em> 12 (10): e0186301. <a href="https://doi.org/10.1371/journal.pone.0186301">https://doi.org/10.1371/journal.pone.0186301</a>.
</div>
<div id="ref-katzman2018" class="csl-entry">
Katzman, J. L., U. Shaham, A. Cloninger, J. Bates, T. Jiang, and Y. Kluger. 2018. <span>“<span>DeepSurv</span>: Personalized Treatment Recommender System Using a <span>Cox</span> Proportional Hazards Deep Neural Network.”</span> <em>BMC Med Res Methodol</em> 18 (1): 24. <a href="https://doi.org/10.1186/s12874-018-0482-1">https://doi.org/10.1186/s12874-018-0482-1</a>.
</div>
<div id="ref-kim2008" class="csl-entry">
Kim, W. R., S. W. Biggins, W. K. Kremers, R. H. Wiesner, P. S. Kamath, J. T. Benson, E. Edwards, and T. M. Therneau. 2008. <span>“Hyponatremia and Mortality Among Patients on the Liver-Transplant Waiting List.”</span> <em>N Engl J Med</em> 359 (10): 1018–26. <a href="https://doi.org/10.1056/NEJMoa0801209">https://doi.org/10.1056/NEJMoa0801209</a>.
</div>
<div id="ref-kvamme2019" class="csl-entry">
Kvamme, H., Ø. Borgan, and I. Scheel. 2019. <span>“Time-to-<span>Event</span> <span>Prediction</span> with <span>Neural</span> <span>Networks</span> and <span>Cox</span> <span>Regression</span>.”</span> <em>J Mach Learn Res</em> 20 (129): 1–30.
</div>
<div id="ref-mlr3" class="csl-entry">
Lang, Michel, Martin Binder, Jakob Richter, Patrick Schratz, Florian Pfisterer, Stefan Coors, Quay Au, Giuseppe Casalicchio, Lars Kotthoff, and Bernd Bischl. 2019. <span>“<span class="nocase">mlr3</span>: A Modern Object-Oriented Machine Learning Framework in <span>R</span>.”</span> <em>Journal of Open Source Software</em>, December. <a href="https://doi.org/10.21105/joss.01903">https://doi.org/10.21105/joss.01903</a>.
</div>
<div id="ref-R-mlr3" class="csl-entry">
Lang, Michel, Bernd Bischl, Jakob Richter, Patrick Schratz, and Martin Binder. 2022. <em>Mlr3: Machine Learning in r - Next Generation</em>. <a href="https://CRAN.R-project.org/package=mlr3">https://CRAN.R-project.org/package=mlr3</a>.
</div>
<div id="ref-malinchoc2000" class="csl-entry">
Malinchoc, M., P. S. Kamath, F. D. Gordon, C. J. Peine, J. Rank, and P. C. J. ter Borg. 2000. <span>“A Model to Predict Poor Survival in Patients Undergoing Transjugular Intrahepatic Portosystemic Shunts.”</span> <em>Hepatology</em> 31 (4): 864–71. <a href="https://doi.org/10.1053/he.2000.5852">https://doi.org/10.1053/he.2000.5852</a>.
</div>
<div id="ref-optn_policies2021" class="csl-entry">
Organ Procurement and Transplantation Network. 2021. <span>“Policies.”</span> 2021. <a href="https://optn.transplant.hrsa.gov/media/1200/optn_policies.pdf">https://optn.transplant.hrsa.gov/media/1200/optn_policies.pdf</a>.
</div>
<div id="ref-ploeg2014" class="csl-entry">
Ploeg, T. van der, P. C. Austin, and E. W. Steyerberg. 2014. <span>“Modern Modelling Techniques Are Data Hungry: A Simulation Study for Predicting Dichotomous Endpoints.”</span> <em><span>BMC</span> Medical Research Methodology</em> 14 (1). <a href="https://doi.org/10.1186/1471-2288-14-137">https://doi.org/10.1186/1471-2288-14-137</a>.
</div>
<div id="ref-R-base" class="csl-entry">
R Core Team. 2022. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-remmler2018" class="csl-entry">
Remmler, J., C. Schneider, T. Treuner-Kaueroff, M. Bartels, D. Seehofer, M. Scholz, T. Berg, and T. Kaiser. 2017. <span>“Increased Level of Interleukin 6 Associates with Increased 90-Day and 1-Year Mortality in Patients with End-Stage Liver Disease.”</span> <em>Clin Gastroenterol Hepatol</em> 16 (5): 730–37. <a href="https://doi.org/10.1016/j.cgh.2017.09.017">https://doi.org/10.1016/j.cgh.2017.09.017</a>.
</div>
<div id="ref-glmnet2011" class="csl-entry">
Simon, Noah, Jerome Friedman, Trevor Hastie, and Rob Tibshirani. 2011. <span>“Regularization Paths for Cox’s Proportional Hazards Model via Coordinate Descent.”</span> <em>Journal of Statistical Software</em> 39 (5): 1–13. <a href="https://doi.org/10.18637/jss.v039.i05">https://doi.org/10.18637/jss.v039.i05</a>.
</div>
<div id="ref-R-gtsummary" class="csl-entry">
Sjoberg, Daniel D., Michael Curry, Joseph Larmarange, Jessica Lavery, Karissa Whiting, and Emily C. Zabor. 2022. <em>Gtsummary: Presentation-Ready Data Summary and Analytic Result Tables</em>. <a href="https://CRAN.R-project.org/package=gtsummary">https://CRAN.R-project.org/package=gtsummary</a>.
</div>
<div id="ref-gtsummary" class="csl-entry">
Sjoberg, Daniel D., Karissa Whiting, Michael Curry, Jessica A. Lavery, and Joseph Larmarange. 2021. <span>“Reproducible Summary Tables with the Gtsummary Package.”</span> <em><span>The R Journal</span></em> 13: 570–80. <a href="https://doi.org/10.32614/RJ-2021-053">https://doi.org/10.32614/RJ-2021-053</a>.
</div>
<div id="ref-R-survivalmodels" class="csl-entry">
Sonabend, Raphael. 2022. <em>Survivalmodels: Models for Survival Analysis</em>. <a href="https://github.com/RaphaelS1/survivalmodels/">https://github.com/RaphaelS1/survivalmodels/</a>.
</div>
<div id="ref-R-mlr3proba" class="csl-entry">
Sonabend, Raphael, Franz Kiraly, and Michel Lang. 2022. <em>Mlr3proba: Probabilistic Supervised Learning for Mlr3</em>.
</div>
<div id="ref-mlr3proba2021" class="csl-entry">
Sonabend, Raphael, Franz J Király, Andreas Bender, Bernd Bischl, and Michel Lang. 2021. <span>“Mlr3proba: An r Package for Machine Learning in Survival Analysis.”</span> <em>Bioinformatics</em>, February. <a href="https://doi.org/10.1093/bioinformatics/btab039">https://doi.org/10.1093/bioinformatics/btab039</a>.
</div>
<div id="ref-survival-book" class="csl-entry">
Terry M. Therneau, and Patricia M. Grambsch. 2000. <em>Modeling Survival Data: Extending the <span>C</span>ox Model</em>. New York: Springer.
</div>
<div id="ref-thabut2007" class="csl-entry">
Thabut, D., J. Massard, A. Gangloff, N. Carbonell, C. Francoz, E. Nguyen-Khac, C. Duhamel, D. Lebrec, T. Poynard, and R. Moreau. 2007. <span>“Model for End-Stage Liver Disease Score and Systemic Inflammatory Response Are Major Prognostic Factors in Patients with Cirrhosis and Acute Functional Renal Failure.”</span> <em>Hepatology</em> 46 (6): 1872–82. <a href="https://doi.org/10.1002/hep.21920">https://doi.org/10.1002/hep.21920</a>.
</div>
<div id="ref-R-survival" class="csl-entry">
Therneau, Terry M. 2022. <em>Survival: Survival Analysis</em>. <a href="https://github.com/therneau/survival">https://github.com/therneau/survival</a>.
</div>
<div id="ref-tibshirani1997" class="csl-entry">
Tibshirani, R. 1997. <span>“The Lasso Method for Variable Selection in the Cox Model.”</span> <em>Stat Med</em> 16 (4): 385–95. <a href="https://doi.org/10.1002/(sici)1097-0258(19970228)16:4&lt;385::aid-sim380&gt;3.0.co;2-3">https://doi.org/10.1002/(sici)1097-0258(19970228)16:4&lt;385::aid-sim380&gt;3.0.co;2-3</a>.
</div>
<div id="ref-trotter2004" class="csl-entry">
Trotter, J. F., B. Brimhall, R. Arjal, and C. Phillips. 2004. <span>“Specific Laboratory Methodologies Achieve Higher Model for Endstage Liver Disease (<span>MELD</span>) Scores for Patients Listed for Liver Transplantation.”</span> <em>Liver Transpl</em> 10 (8): 995–1000. <a href="https://doi.org/10.1002/lt.20195">https://doi.org/10.1002/lt.20195</a>.
</div>
<div id="ref-wiesner2003" class="csl-entry">
Wiesner, R., E. Edwards, R. Freeman, A. Harper, R. Kim, P. Kamath, W. Kremers, et al. 2003. <span>“Model for End-Stage Liver Disease (<span>MELD</span>) and Allocation of Donor Livers.”</span> <em>Gastroenterology</em> 124 (1): 91–96. <a href="https://doi.org/10.1053/gast.2003.50016">https://doi.org/10.1053/gast.2003.50016</a>.
</div>
<div id="ref-R-ranger" class="csl-entry">
Wright, Marvin N., Stefan Wager, and Philipp Probst. 2021. <em>Ranger: A Fast Implementation of Random Forests</em>. <a href="https://github.com/imbs-hl/ranger">https://github.com/imbs-hl/ranger</a>.
</div>
<div id="ref-ranger2017" class="csl-entry">
Wright, Marvin N., and Andreas Ziegler. 2017. <span>“<span class="nocase">ranger</span>: A Fast Implementation of Random Forests for High Dimensional Data in <span>C++</span> and <span>R</span>.”</span> <em>Journal of Statistical Software</em> 77 (1): 1–17. <a href="https://doi.org/10.18637/jss.v077.i01">https://doi.org/10.18637/jss.v077.i01</a>.
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.0 (2022-04-22)
Platform: x86_64-unknown-linux-gnu (64-bit)

Matrix products: default
BLAS/LAPACK: /gnu/store/ras6dprsw3wm3swk23jjp8ww5dwxj333-openblas-0.3.18/lib/libopenblasp-r0.3.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] gtsummary_1.6.0   timeROC_0.4       ameld_0.0.26      survival_3.3-1   
 [5] glmnet_4.1-4      Matrix_1.4-1      viridis_0.6.2     viridisLite_0.4.0
 [9] mlr3viz_0.5.9     ggplot2_3.3.6     targets_0.12.1   

loaded via a namespace (and not attached):
  [1] fs_1.5.2                 RColorBrewer_1.1-3       bbotk_0.5.3             
  [4] rprojroot_2.0.3          mlr3pipelines_0.4.1      mlr3proba_0.4.11        
  [7] numDeriv_2016.8-1.1      tools_4.2.0              backports_1.4.1         
 [10] bslib_0.3.1              utf8_1.2.2               R6_2.5.1                
 [13] mlr3_0.13.3              colorspace_2.0-3         withr_2.5.0             
 [16] mlr3misc_0.10.0          tidyselect_1.1.2         gridExtra_2.3           
 [19] processx_3.5.3           compiler_4.2.0           git2r_0.30.1            
 [22] cli_3.3.0                ooplah_0.2.0             gt_0.6.0                
 [25] lgr_0.4.3                bookdown_0.26            sass_0.4.1              
 [28] scales_1.2.0             checkmate_2.1.0          mvtnorm_1.1-3           
 [31] pec_2022.05.04           callr_3.7.0              palmerpenguins_0.1.0    
 [34] commonmark_1.8.0         mlr3tuning_0.13.1        stringr_1.4.0           
 [37] digest_0.6.29            mlr3extralearners_0.5.37 rmarkdown_2.14          
 [40] param6_0.2.4             paradox_0.9.0            set6_0.2.4              
 [43] pkgconfig_2.0.3          htmltools_0.5.2          parallelly_1.31.1       
 [46] highr_0.9                fastmap_1.1.0            htmlwidgets_1.5.4       
 [49] rlang_1.0.2              rstudioapi_0.13          visNetwork_2.1.0        
 [52] shape_1.4.6              jquerylib_0.1.4          generics_0.1.2          
 [55] jsonlite_1.8.0           dplyr_1.0.9              magrittr_2.0.3          
 [58] Rcpp_1.0.8.3             munsell_0.5.0            fansi_1.0.3             
 [61] lifecycle_1.0.1          stringi_1.7.6            whisker_0.4             
 [64] yaml_2.3.5               grid_4.2.0               parallel_4.2.0          
 [67] dictionar6_0.1.3         listenv_0.8.0            promises_1.2.0.1        
 [70] crayon_1.5.1             lattice_0.20-45          splines_4.2.0           
 [73] knitr_1.39               ps_1.7.0                 pillar_1.7.0            
 [76] timereg_2.0.2            igraph_1.3.1             uuid_1.1-0              
 [79] base64url_1.4            future.apply_1.9.0       codetools_0.2-18        
 [82] glue_1.6.2               evaluate_0.15            broom.helpers_1.7.0     
 [85] data.table_1.14.2        vctrs_0.4.1              httpuv_1.6.5            
 [88] foreach_1.5.2            distr6_1.6.9             tidyr_1.2.0             
 [91] gtable_0.3.0             purrr_0.3.4              future_1.26.1           
 [94] xfun_0.31                prodlim_2019.11.13       later_1.3.0             
 [97] tibble_3.1.7             iterators_1.0.14         workflowr_1.7.0         
[100] lava_1.6.10              DiagrammeR_1.0.9         globals_0.15.0          
[103] ellipsis_0.3.2          </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
